_modules/conditionals.md

```diff
@@ -6,3 +6,119 @@ short_title: Buttons and Switches
 icon: fa-solid fa-toggle-on
 order: 6
 ---
+
+## What are Conditionals?
```

Comment:
I think we need to introduce the button, switch and knob functions first?

Reply:
Is it okay to introduce the knob function at the start of module 7, before they design the badge? Since it carries a discrete set of values, I think it's a little harder to make it a conditional in a practical way.

Reply:
Done